src/components/Header/TopMainNav/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useState } from 'react';
+import cn from 'classnames';
```

Comment:
i think `import cn from 'classnames';` is not being used in this file component anymore?